flow/flow_ebos_onephase_energy.cpp

```diff
@@ -60,7 +60,34 @@ struct Indices<TypeTag, TTag::EclFlowProblemSimple>
 
 } // namespace Opm::Properties
 
-int flowEbosOnephaseEnergyMain(int argc, char** argv)
+void flowEbosWaterEnergyOnlySetDeck(double setupTime, std::shared_ptr<Deck> deck,
```

Comment:
Shouldn't be 
` void flowEbosWaterOnlyEnergySetDeck`
instead of 
` void flowEbosWaterEnergyOnlySetDeck`
?

Reply:
Indeed. Good catch.